CalibCalorimetry/EcalLaserCorrection/python/ecalLaserCorrectionServiceMC_cfi.py

```diff
@@ -0,0 +1,3 @@
+import FWCore.ParameterSet.Config as cms
+
+EcalLaserCorrectionServiceMC = cms.ESProducer("EcalLaserCorrectionServiceMC")
```

Comment:
The `_cfi` file should be left to be generated automatically with the fillDescriptions()
See https://cms-sw.github.io/cms_coding_rules.html and https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#Converting_an_Existing_CFI_file

Reply:
The latest commit (4147e8f) makes this change.